### PR TITLE
Feature/role switch & mission

### PR DESCRIPTION
**Pull request by bbinoy**

### CODE REVIEW

File: src/locale/fr/messages.po [hunk 1]
```diff
@@ -1405,14 +1345,14 @@ msgstr "Les visualisations t’aident à comprendre ce qu’il se passe"
 msgid "Want to model the new financing round for your company? Use Ledgy!"
 msgstr "Tu veux modéliser la nouvelle levée de fonds pour ton entreprise? Utilise Ledgy!"
 
+#: src/pages/about-us.jsx:91
+msgid "We believe that entrepreneurship is the main driver of positive change in the world. That is why we build beautiful and easy to use software for startups."
+msgstr "Nous croyons que l'entrepreneuriat est le principal moteur du changement positif dans le monde. C'est pourquoi nous construisons des logiciels beaux et faciles à utiliser pour les startups."
```
Comment (koyanoa) [inline src/locale/fr/messages.po]:
```suggestion
msgstr "Nous croyons que l'entrepreneuriat est le principal moteur de changement positif dans le monde. C'est pourquoi nous construisons des logiciels intuitifs et faciles à utiliser pour les startups."
```

**Comment by koyanoa** [comment]
image
ups, this doesn't look nice

File: src/pages/about-us.jsx [hunk 2]
```diff
@@ -208,12 +196,6 @@ export const pageQuery = graphql`
   query {
     ...TeamFragment
 
-    mission: imageSharp(fluid: { originalName: { regex: "/mission/" } }) {
```
Comment (morloy) [inline src/pages/about-us.jsx]: Please also delete the corresponding file, not only the reference.

Comment (bbinoy) [reply src/pages/about-us.jsx]: I thinnk I did?
image

**Pull request merged**